src/fitnesse/wiki/fs/WikiFilePage.java

```diff
@@ -144,7 +144,7 @@ private PageData getDataVersion() throws IOException {
 
     if (fileVersion != null) {
       try {
-        String fileContent = loadContent(fileVersion);
+        String fileContent = loadContent(fileVersion).replace("\r\n", "\n");
 
```

Comment:
May I suggest using `replaceAll("\\r\\n", "\n")` as to ensure the regex grabs the line endings properly as replace uses literal matches?